snuba/datasets/configuration/generic_metrics/entities/org_counters.yaml

```diff
@@ -0,0 +1,57 @@
+version: v1
+kind: entity
+name: org_metrics_counters
+schema:
+  [
+    { name: org_id, type: UInt, args: { size: 64 } },
+    { name: project_id, type: UInt, args: { size: 64 } },
+    { name: metric_id, type: UInt, args: { size: 64 } },
+    { name: timestamp, type: DateTime },
+    { name: bucketed_time, type: DateTime },
+  ]
+required_time_column: timestamp
+
+storages:
+  - storage: org_metrics_counters
```

Comment:
I mentioned this in Slack, but this storage is referencing the release health metrics counters. You will need a new storage in `generic_metrics/storages/` that will be very similar to [this one](https://github.com/getsentry/snuba/blob/master/snuba/datasets/configuration/metrics/storages/org_counters.yaml), but with a new name. Then you can reference that storage here.